src/commands/utils/encrypt.ts

```diff
@@ -0,0 +1,30 @@
+import { Account, Nevermined } from '@nevermined-io/nevermined-sdk-js'
+import { ExecutionOutput } from '../../models/ExecutionOutput'
+import fs from 'fs'
+import { Logger } from 'log4js'
+import { StatusCodes } from '../../utils'
+import { ConfigEntry } from '../../models/ConfigDefinition'
+import { aes_encryption_256 } from '@nevermined-io/nevermined-sdk-dtp/dist/utils'
+
+export const encryptFile = async (
+  nvm: Nevermined,
+  account: Account,
+  argv: any,
+  config: ConfigEntry,
+  logger: Logger
+): Promise<ExecutionOutput> => {
+  const { file, password } = argv
+  const encrypted = fs.readFileSync(file).toString('binary')
+  const decrypted = aes_encryption_256(encrypted, password)
+
+  const filePathEncrypted = file + '.encrypted'
```

Comment:
Similar to other commands here we can add an intro message saying what we are doing